### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by anedosugov**

### CODE REVIEW

**Comment by anedosugov** [comment]
Keyspaces logo still missing, otherwise done

File: src/components/DynamoDB.vue [hunk 1]
```diff
@@ -1,70 +1,105 @@
...
+      <div>
+        <div class="price-name text-capitalize">
+          {{ price.name }}
+          <div class="font-weight-light d-inline-block">
+            {{ price.subname }}
+          </div>
+        </div>
+        <div class="price__wrapper">
+          <div class="price d-flex align-items-baseline">
+            <small>$</small
```
Comment (nukemberg) [inline src/components/DynamoDB.vue:13]: why this instead of `currency` style?

File: src/components/ScyllaCloud.vue [hunk 2]
```diff
@@ -59,7 +67,7 @@
           </table>
         </div>
       </div>
-      <div class="tech-specs">
+      <div v-if="!hideSpecs" class="tech-specs">
```
Comment (nukemberg) [inline src/components/ScyllaCloud.vue]: use `v-show`

File: src/Calculator.vue [hunk 3]
```diff
@@ -1,182 +1,29 @@
...
+            :disabled="slider.disabled"
+            @update="$data.workload[slider.title] = $event"
+          ></SliderInput>
         </form>
       </div>
       <div class="col-md-7 col-sm-12 right-column d-flex flex-column">
         <div class="right-column__header d-flex">
-          <div class="billing-toggle-wrapper d-flex align-items-center">
+          <div
+            v-if="selectedDropdownItem.name === 'details'"
```
Comment (nukemberg) [inline src/Calculator.vue]: use `v-show`

File: src/Calculator.vue [hunk 4]
```diff
@@ -186,36 +33,93 @@
...
+                  :is="calc1"
+                  v-model="scyllaPrices"
+                  :workload="workload"
+                  :reserved="true"
+                  hide-specs
+                ></component>
+              </div>
+              <div class="col-6 pl-3 pr-0">
+                <div class="calculator-header">
+                  <img
```
Comment (nukemberg) [inline src/Calculator.vue:92]: branching for every possible image is not nice. pass an image name parameter instead, i don't want N img tags with if statements

**Review by nukemberg — CHANGES_REQUESTED** [review]
The way you are using calculator components multiple times would make sense if all they did was simply display data - but they also compute data; if you want to use them like that refactor the computation part out and have them as display only components which receive the results of the computation. the original reason i coupled the calculator model and display was because models may have additional specific inputs (e.g. Astra tier), but this can be handled by breaking the components to inputs and outputs

File: src/components/DynamoDB.vue (hunk 1 above)
Comment (anedosugov) [reply src/components/DynamoDB.vue:13]: more control over styling and :
Screenshot 2021-01-12 at 11 04 45

File: src/Calculator.vue (hunk 3 above)
Comment (anedosugov) [reply src/Calculator.vue]: fixed

File: src/Calculator.vue (hunk 4 above)
Comment (anedosugov) [reply src/Calculator.vue:92]: fixed, ran into issue with webpack initially

File: src/components/ScyllaCloud.vue (hunk 2 above)
Comment (anedosugov) [reply src/components/ScyllaCloud.vue]: makes no sense to render it and hide it if the prop is set to hide the specs for this component instance (won't be switched back and forth shown during its lifetime in our use case)

**Comment by anedosugov** [comment]
> The way you are using calculator components multiple times would make sense if all they did was simply display data - but they also compute data; if you want to use them like that refactor the computation part out and have them as display only components which receive the results of the computation. the original reason i coupled the calculator model and display was because models may have additional specific inputs (e.g. Astra tier), but this can be handled by breaking the components to inputs and outputs

Makes sense total to refactor, and I could give it a shot, however as I'm not an expert TS user I don't know how long it will take.  Also the task assigned was styling only..

**Pull request merged**